### ISSUE TITLE
Make English Doc.

### ISSUE THREAD

**Issue by alphaelf** [museui/muse-ui#100]
hi, in all of material design pack for vue 2, this is number 1 but documentation not readable much more

is possible for you to make this [https://museui.github.io]() to english?

**Comment by linxiaowu66** [comment]
@alphaelf 
   Thanks for your attention. We has already prepared for it.

**Comment by donnygunawan93** [comment]
yeah, make components in english also, issues conversation in english also, xiexie hahaha.

**Comment by Jordy-A** [comment]
+1!

**Comment by insign** [comment]
+1

**Comment by ClaudioEyzaguirre** [comment]
+1

**Comment by 0xFireball** [comment]
+1!

**Comment by myronliu347** [comment]
Some have a document in English now

**Comment by myronliu347** [comment]
We are in the translation

**Comment by insign** [comment]
Can we help? (I do not know chinese)

*Hélio Araújo de Oliveira*
Evite correntes de e-mails e combata o spam.
Prefira materiais reciclados, incluindo papel

2016-12-19 22:32 GMT-03:00 myron <notifications@github.com>:

**Comment by alphaelf** [comment]
@myronliu347  thanks a lot.
